src/Symfony/Component/Yaml/Dumper.php

```diff
@@ -42,17 +44,23 @@ public function setIndentation($num)
      * @param int   $inline                 The level where you switch to inline YAML
      * @param int   $indent                 The level of indentation (used internally)
      * @param bool  $exceptionOnInvalidType true if an exception must be thrown on invalid types (a PHP resource or object), false otherwise
-     * @param bool  $objectSupport          true if object support is enabled, false otherwise
+     * @param int   $flags                  A bit field of DUMP_* constants to customize the dumped YAML string
```

Comment:
Bit is right. Each future dumper option will get its own bit so that we can use PHP's bitwise operators to check if a particular dumper option was requested.